packages/ui/src/components/molecules/Send/FeesSection.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import SvgGoldQuestionMark from '../../../assets/icons/generated/GoldQuestionMark';
```

Comment:
Change this in all other places

```suggestion
import { GoldQuestionMark } from '../../../assets';
```

Reply:
done